docs/creators-content-guidelines.md

```diff
@@ -2,4 +2,5 @@
 id: creators-content-guidelines
 title: Featured Content Guidelines
 sidebar_label: Featured Content Guidelines
+description: Placeholder
```

Comment:
It might work to set the description to undefined or null, but otherwise this shouldn't be set, so the incipit will be used.